OJP/OJP_Trips.xsd

```diff
@@ -631,59 +631,219 @@
 			</xs:element>
 			<xs:element name="GuidanceAdvice" type="GuidanceAdviceEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>various types of guidance advice given to travelle.</xs:documentation>
+					<xs:documentation>Various types of guidance advice given to traveller.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="TurnAction" type="TurnActionEnumeration" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>the range of alternative turns that can be described.</xs:documentation>
+					<xs:documentation>The range of alternative turns that can be described.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="RoadName" type="InternationalTextStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Road name, if applicable.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Follow" type="FollowStructure" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Signs, roads, POI to follow.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="DirectionHint" type="InternationalTextStructure" minOccurs="0">
 				<xs:annotation>
 					<xs:documentation>Textual direction hint for better understanding, e.g. "follow signs to Hamburg".</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="Bearing" type="siri:AbsoluteBearingType" minOccurs="0">
+			<xs:element name="AbsoluteBearing" type="siri:AbsoluteBearingType" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Absolute bearing after the described manoeuvre.</xs:documentation>
+					<xs:documentation>Absolute bearing (sky direction) after the described manoeuvre.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="PathLink" type="PathLinkStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Description of the type of accessibility on this navigation section.</xs:documentation>
+					<xs:documentation>Description of the type of accessibility on this navigation section. This is greatly reduced from the NeTEx PathLink structure.</xs:documentation>
```

Comment:
"This is greatly reduced from the NeTEx PathLink structure." => "This is greatly reduced in comparison to the NeTEx PathLink structure."?

Reply:
rewrote it